debian/chromium-browser.sh.in

```diff
@@ -7,6 +7,13 @@
 #  Fabien Tassin <fta@sofaraway.org>
 # License: GPLv2 or later
 
+ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+
+if ! malcontent-client check ${ABSOLUTE_PATH} >/dev/null; then
```

Comment:
These two review comments haven’t been addressed:

Might as well add a `--quiet` option to malcontent-client check upstream so that the output redirection isn’t necessary.

And you should probably pass `--no-interactive` to avoid the possibility of it raising polkit authentication dialogues. Unless that’s the desired behaviour? In which case maybe pass `--interactive` to make that explicit.

Reply:
@pwithnall Oops, missed these 2 - I think adding --quiet is a bit out of scope for this task - could you please add a new task for that? 

I will update it to pass --non-interactive.

Reply:
Updated to pass --no-interactive, left the redirection to >/dev/null for now - may add it to malcontent when working on supporting disabling apps by content type.

Reply:
I’ve [just added it to malcontent upstream](https://gitlab.freedesktop.org/pwithnall/malcontent/merge_requests/13). You’ll need to double-check my patch and then update [our fork](https://github.com/endlessm/malcontent/) (which should be a simple cherry pick or merge) and bump the `malcontent-tools` dependency added in this PR.

Use it as `malcontent-client --quiet check ${ABSOLUTE_PATH}`.

Reply:
(It seems silly to avoid doing this 10 minutes of work now, in favour of carrying the same technical debt in this package *and* Chrome.)

Reply:
Updated again to support --quiet.